### PR TITLE
Convert into a Python package

### PR DESCRIPTION
**Pull request by apdavison**
so that people will be able to install it with pip, once we release the first version.

For development, you can install with `pip install -e .`, then to run the converter:
```
$ python -m bids2openminds.converter
```

### CODE REVIEW

**Comment by Peyman-N** [comment]
There is also another import in mapping.py
I think that one should be modified as well 
`  from utility import openminds_instance`

**Comment by apdavison** [comment]
Good spot. I've fixed that, and moved all imports to the top of each file.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[project]
+name = "bids2openminds"
+version = "0.0.0"
+dependencies = [
+  "bids",
+  "openminds",
```
Comment (Peyman-N) [inline pyproject.toml:6]: I think we should add a openminds version as only the latest one can use instances 
`"openminds >= 0.2.0 ",`

Comment (lzehl) [reply pyproject.toml:6]: More important: v2 and v1 have errors. so we should use the bids2openminds only for openminds v3 and higher

Comment (apdavison) [reply pyproject.toml:6]: @lzehl the target openMINDS version could be added as a command-line option: @Peyman-N could you create an issue for that?

For the dependency on openMINDS-Python, I'm making a new release 0.2.1 which includes the improved IRI validation.

Comment (apdavison) [reply pyproject.toml]: ok, @Peyman-N that's done (>= 0.2.1)

**Pull request merged**